notebook/modelling/n-queens.ipynb

```diff
@@ -4,97 +4,113 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## N-Queens\n",
-    "\n",
-    "**Originally Contributed by:** Matthew Helm\n",
-    "\n",
-    "The N-Queens problem involves placing N queens on an N x N chessboard such that none of the queens attacks another. In chess, a queen can move vertically, horizontally, and diagonally so there cannot be more than one queen on any given row, column, or diagonal."
+    "# N-Queens"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "![](https://raw.githubusercontent.com/mthelm85/nqueens/master/n_queens4.png)"
+    "**Originally Contributed by**: Matthew Helm ([with help from @mtanneau on Julia Discourse](https://discourse.julialang.org/t/which-jump-jl-solver-for-this-problem/43350/17?u=mthelm85))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "*Note that none of the queens above are able to attack any other as a result of their careful placement.*"
+    "The N-Queens problem involves placing N queens on an N x N chessboard such that none of the queens attacks another. In chess, a \n",
+    "queen can move vertically, horizontally, and diagonally so there cannot be more than one queen on any given row, column, or \n",
+    "diagonal."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 1,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "using GLPK\n",
-    "using JuMP\n",
-    "using LinearAlgebra"
+    "<img src=\"img/n_queens4.png\" style=\"width: auto; height: auto\" alt=\"4 Queens\">"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 2,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "8"
-      ]
-     },
-     "execution_count": 2,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
    "source": [
-    "# N-Queens\n",
-    "N = 8"
+    "*Note that none of the queens above are able to attack any other as a result of their careful placement.*"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "┌ Info: Precompiling GLPK [60bf3e95-4087-53dc-ae20-288a0d20c6a6]\n",
+      "└ @ Base loading.jl:1192\n",
+      "ERROR: LoadError: GLPK not properly installed. Please run Pkg.build(\"GLPK\")\n",
+      "Stacktrace:\n",
+      " [1] error(::String) at ./error.jl:33\n",
+      " [2] top-level scope at /home/mbesancon/.julia/packages/GLPK/oTTtu/src/GLPK.jl:197\n",
```

Comment:
@matbesancon

Reply:
ah damn, sorry about that (this is surprising that the build errors silently when things break like this)

Reply:
@mthelm85 could you rerun the generation of this notebook?

Reply:
Done